MMR.Randomizer/Builder.cs

```diff
@@ -120,36 +102,37 @@ void WriteOutput(string str)
                         continue;
 
                     // test if the testSeq can be used with available instrument set slots
-                    if (testSeq.SequenceBinaryList != null  && testSeq.SequenceBinaryList.Count > 0 && testSeq.SequenceBinaryList[0].InstrumentSet != null)
+                    if (testSeq.SequenceBinaryList != null && testSeq.SequenceBinaryList.Count > 0 && testSeq.SequenceBinaryList[0].InstrumentSet != null)
                     {
                         // randomize instrument sets last second, so the early banks don't get ravaged based on order
                         if (testSeq.SequenceBinaryList.Count > 1)
                             testSeq.SequenceBinaryList.OrderBy(x => random.Next()).ToList();
```

Comment:
This line doesn't do anything. .ToList() creates a new array, and then discards it since it doesn't assign it to anything.

Reply:
Yep, looks like I forgot to re-assign the result to the original list, kinda surprised intelisense didn't warn that the result was discarded.